syncano/models/options.py

```diff
@@ -95,7 +95,7 @@ def add_field(self, field):
             raise SyncanoValueError('Field "{0}" already defined'.format(field.name))
 
         self.field_names.append(field.name)
-        self.fields.append(field)
+        self.fields.insert(field.creation_counter, field)
```

Comment:
This doesn't seem right. 
E.g. 
self.fields.insert(5, field1)
self.fields.insert(3, field2)
will cause fields to contin [field1,field2] instead of expected [field2, field1].
unless self.fields was initialized with proper length before.

Besides, isn't this add_field called after the initialization of each field? So it will be executed sequentially in order of fields anyway? And anyway, why do you care about the order here?